### PR TITLE
[PBW-5158] resolve - Cannot read 'addEventListener' of undefined

### PR DESCRIPTION
**Pull request by mistermendez**
Resolve - Uncaught exception TypeError: Cannot read property 'addEventListener' of undefined

### CODE REVIEW

File: js/controller.js [hunk 1]
```diff
@@ -269,7 +269,10 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
         element = elementVideo;
```
Comment (pilievOoyala) [inline js/controller.js]: Do you need to do any check if "element" exists? A lot of the function depends on it. Also, do you need to do something else if element or element.get(0) doesn't exist or just let it continue on?

Comment (mistermendez) [inline js/controller.js]: * yup just let it continue, we can't add event listener w/o main video element

Comment (chibzu) [inline js/controller.js]: Isn't the point of this event that the element should exist? Do we understand why we're getting an event without the element being instantiated?

Comment (pilievOoyala) [inline js/controller.js]: I've already talked to Angie about it. I don't recall what the final outcome was, but there is already a ticket for why it's not instantiated.  So this checkin should just be concerned with what to do if we get bad data and being able to fail gracefully.

**Comment by chibzu** [comment]
If that's the case, this will fail gracefully. Since the element is grabbed with JQuery it's wrapped so it won't throw exceptions even though there isn't an element there.

**Comment by chibzu** [comment]
I believe the concerns here have been addressed, lgtm.

**Pull request merged**